src/diff.test.ts

```diff
@@ -35,7 +35,7 @@ test('section', () => {
         'bundles',
       )} got bigger`,
       isEmpty: diff.chunks.bigger.length === 0,
-      children: renderBiggerTable({ assets: diff.chunks.bigger }),
```

Comment:
**suggestion**: it would be useful to add a unit test to capture the new functionality, especially for new contributors.